tools/cldr-unittest/src/org/unicode/cldr/unittest/TestExampleGenerator.java

```diff
@@ -453,9 +451,9 @@ public void TestCurrencyFormats() {
         String actual = simplify(exampleGenerator
             .getExampleHtml(
                 "//ldml/numbers/currencyFormats[@numberSystem=\"latn\"]/currencyFormatLength/currencyFormat[@type=\"standard\"]/pattern[@type=\"standard\"]",
-                "¤ #0.00", ExampleType.ENGLISH));
+                "¤ #0.00"));
         assertEquals("Currency format example faulty",
-            "〖USD ❬1295,00❭〗〖-USD ❬1295,00❭〗", actual);
+            "〖€ ❬1295,00❭〗〖-€ ❬1295,00❭〗", actual);
```

Comment:
The test needs to be changed so that the currency is USD, so that the same results are obtained as before. (If it can't be so changed, then there is a likely problem.

Reply:
The original test is:
`    public void TestCurrencyFormats() {
        ExampleGenerator exampleGenerator = getExampleGenerator("it");
        String actual = simplify(exampleGenerator
            .getExampleHtml(
                "//ldml/numbers/currencyFormats[@numberSystem=\"latn\"]/currencyFormatLength/currencyFormat[@type=\"standard\"]/pattern[@type=\"standard\"]",
                "¤ #0.00", ExampleType.ENGLISH));
        assertEquals("Currency format example faulty",
            "〖USD ❬1295,00❭〗〖-USD ❬1295,00❭〗", actual);
    }
`
In SurveyTool, ExampleType.ENGLISH is never used with a non-English locale like "it". What is the test really supposed to be testing? What is the basis for expecting "USD" for an Italian ExampleGenerator?

Reply:
Ah, ok. Given that context, it's ok to change the test.